assets/js/components/SapSystemDetails/tableConfigs.jsx

```diff
@@ -1,29 +1,84 @@
 import React from 'react';
+import classNames from 'classnames';
 
 import HostLink from '@components/HostLink';
 import ProviderLabel from '@components/ProviderLabel';
+import CleanUpButton from '@components/CleanUpButton';
+import Tooltip from '@components/Tooltip';
+import HealthIcon from '@components/Health';
 
 import Features from './Features';
 import InstanceStatus from './InstanceStatus';
 
+const cellRender = (content, item) => (
+  <span className={classNames({ 'text-gray-600': !!item.absent_at })}>
+    {content}
+  </span>
+);
+
 export const systemInstancesTableConfiguration = {
   usePadding: false,
   columns: [
-    { title: 'Hostname', key: 'instance_hostname' },
-    { title: 'Instance Number', key: 'instance_number' },
+    {
+      title: 'Hostname',
+      key: 'instance_hostname',
+      render: (content, item) => {
+        if (item.absent_at) {
+          return (
+            <span className="flex items-center">
+              <Tooltip content="Registered instance not found." place="bottom">
+                <HealthIcon health="absent" />
+              </Tooltip>
+              <span className="ml-1 text-gray-600">{content}</span>
+            </span>
+          );
+        }
+        return <span>{content}</span>;
+      },
+    },
+    {
+      title: 'Instance nr',
+      key: 'instance_number',
+      render: cellRender,
+    },
     {
       title: 'Features',
       key: 'features',
       render: (content) => <Features features={content} />,
     },
-    { title: 'Http Port', key: 'http_port' },
-    { title: 'Https Port', key: 'https_port' },
-    { title: 'Start Priority', key: 'start_priority' },
+    {
+      title: 'Http Port',
+      key: 'http_port',
+      render: cellRender,
+    },
+    {
+      title: 'Https Port',
+      key: 'https_port',
+      render: cellRender,
+    },
+    {
+      title: 'Start Prio',
```

Comment:
For some reason you changed `Priority` by `Prio`. I typo I guess hehe

Reply:
I noticed that it was written like that on the Figma link within the task in Jira, I guess the logic here is that it's a common abbreviation and saves us some space.